src/Utils.js

```diff
@@ -7,9 +7,18 @@ import {
 } from './react-compat';
 import {
   REACT013,
-  REACT014,
+  REACT15,
 } from './version';
 
+function internalInstanceKey(node) {
+  return Object.keys(node).filter(key => key.match(/^__reactInternalInstance\$/))[0];
```

Comment:
Per https://travis-ci.org/airbnb/enzyme/jobs/114854317 - pre-ES6, `Object.keys` throws on primitive values. You can either use the `object-keys` module, or do `Object.keys(Object(node))` (but that runs decently slower).